src/modules/BootSupportCheck.rb

```diff
@@ -185,12 +185,12 @@ def check_zipl_part
       boot_part = Storage.GetEntryForMountpoint("/boot") if boot_part.empty?
       boot_part = Storage.GetEntryForMountpoint("/") if boot_part.empty?
 
-      if [:ext2, :ext3, :ext4].include? boot_part["used_fs"]
-        return true
-      else
+      if ![:ext2, :ext3, :ext4].include? boot_part["used_fs"]
```

Comment:
I also considering it, but on other hand, for me it is more natural work-flow in check method, to verify that no problem happend and if nothing found, then return true.